nosqlbench/http-jsonapi-search-basic.yaml

```diff
@@ -1,4 +1,4 @@
-min_version: "5.17.1"
+min_version: "5.17.3"
```

Comment:
Does not run with `nb5 5.17.3-release`:

```
$ nb5 -v --report-csv-to logs --log-histograms logs/hdr_metrics.log http-jsonapi-search-basic jsonapi_host=localhost docscount=10000 connections=25 rate=250
     742 INFO  [main] NBCLI        Configured scenario log at logs/scenario_20230502_132422_664.log
     750 INFO  [main] NBCLI        Running NoSQLBench Version 5.17.3-release
     751 INFO  [main] NBCLI        command-line: -v --report-csv-to logs --log-histograms logs/hdr_metrics.log http-jsonapi-search-basic jsonapi_host=localhost docscount=10000 connections=25 rate=250
    2657 INFO  [main] NBCLI        client-hardware: {"processor-name":"Intel(R) Core(TM) i7-10750H CPU @ 2.60GHz","heap-max-GiB":"7.78","if-speeds":["0Mib","10Gib"],"physical-cores":"6","max-frequency-ghz":"0.01","logical-cores":"12","sockets":"1","memory-GiB":"31.10"}
    2848 INFO  [main] OpsLoader    workload URI: 'file:///home/ise/workspace/datastax/stargate/jsonapi/nosqlbench/http-jsonapi-search-basic.yaml'
    2911 ERROR [main] ERRORHANDLER internal error: class java.util.LinkedHashMap cannot be cast to class java.lang.String (java.util.LinkedHashMap and java.lang.String are in module java.base of loader 'bootstrap')
java.lang.ClassCastException: class java.util.LinkedHashMap cannot be cast to class java.lang.String (java.util.LinkedHashMap and java.lang.String are in module java.base of loader 'bootstrap')
	at io.nosqlbench.engine.api.scenarios.NBCLIScenarioParser.parseScenarioCommand(NBCLIScenarioParser.java:157)
	at io.nosqlbench.engine.cli.NBCLICommandParser.parse(NBCLICommandParser.java:87)
	at io.nosqlbench.engine.cli.NBCLIOptions.parseAllOptions(NBCLIOptions.java:631)
	at io.nosqlbench.engine.cli.NBCLIOptions.<init>(NBCLIOptions.java:245)
	at io.nosqlbench.engine.cli.NBCLIOptions.<init>(NBCLIOptions.java:236)
	at io.nosqlbench.engine.cli.NBCLI.applyDirect(NBCLI.java:248)
	at io.nosqlbench.engine.cli.NBCLI.apply(NBCLI.java:114)
	at io.nosqlbench.engine.cli.NBCLI.main(NBCLI.java:94)
Scenario stopped due to error. See logs for details.

```

Reply:
This scenario tests functionality like `gt` and `lt` that is not implemented yet. I will add a comment to clarify this is not a working scenario yet.

Reply:
have a look at the stack trace, it has nothing to do with tests, seems YAML is wrongs:

```
 2911 ERROR [main] ERRORHANDLER internal error: class java.util.LinkedHashMap cannot be cast to class java.lang.String (java.util.LinkedHashMap and java.lang.String are in module java.base of loader 'bootstrap')
java.lang.ClassCastException: class java.util.LinkedHashMap cannot be cast to class java.lang.String (java.util.LinkedHashMap and java.lang.String are in module java.base of loader 'bootstrap')
```

Reply:
I have updated the scenario to contain a new TODO note, and updated the scenario definitions to correctly reference operations they are intended to run. If you run the default scenario now it will run to completion, correctly indicating the calls that had error messages due to syntax that is not yet implemented.